fusedrug_examples/interaction/drug_target/affinity_prediction/t5_mol/tokenizer/multi_tokenizer.py

```diff
@@ -0,0 +1,1304 @@
+from typing import Dict
+from collections.abc import Iterable
+from fuse.utils import NDict
+from fuse.data import OpBase, get_sample_id
+from tokenizers import Tokenizer, Encoding
+import tokenizers
+from warnings import warn
+from typing import Optional, List, Set, Union, Tuple, Any, Iterator
+import json
+import transformers
+
+
+class ModularTokenizer(transformers.PreTrainedTokenizerBase):
+    def __init__(
+        self,
+        tokenizers_info: List,
+        load_adjusted_jsons: Optional[bool] = False,
+        **kwargs: Any,
+    ) -> None:
+        """_summary_
+
+        Args:
+            tokenizers_info (List): A list of dictionaries containing the following:
+            [
+                   {
+                        "name": a name of a tokenizer
+                        "modular_json_path":out_path for tokenizer_type
```

Comment:
This requires some careful adjustments and tests. Maybe make an issue out of this?

Reply:
Yes, I think, looking forward, that it will be helpful to load tokenizer from from file/dir instead of specifying the entire config.
Please open trello card in core board then.

Reply:
Done